src/neuroconv/datainterfaces/ecephys/neuroscope/neuroscopedatainterface.py

```diff
@@ -127,6 +129,16 @@ def get_metadata(self) -> dict:
             metadata["NWBFile"]["session_start_time"] = session_start_time
         return metadata
 
+    def get_original_timestamps(self) -> np.ndarray:
```

Comment:
3 debugs were necessary to interfaces that alias SI/neo from the NeuroConv `source_data` `__init__` signature

Specifically https://github.com/catalystneuro/neuroconv/pull/459/files#diff-5939435a8408da1ec7aba7f1b54b9eb49747778cc8376f1bac76e9c6fadc06b8R133 differs from the usual

Reply:
Is there a better approach to take overall? Probably but would take a lot more work to refactor every interface to utilize so just doing this spot fix for now

Reply:
I see. Fair enough. Exactly, I was wondering what exactly break and why did the general method logic did not work. In this case, do I understand well that is the variety of initialization signatures require specialized logic? 

If that's the case I think this makes sense. Let's have something that passess all the tests and then refactor to generalize later if possible. Can you add a small comment or docstring just mentioning this. Something like: "necessary to overide method because of different siganture for this extractor in spikeinterface` just so people know what we are overiden the base method here.